manifests/t/TranslucentTB/TranslucentTB/2023.1/TranslucentTB.TranslucentTB.installer.yaml

```diff
@@ -0,0 +1,46 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: TranslucentTB.TranslucentTB
+PackageVersion: "2023.1"
+UpgradeBehavior: install
+ReleaseDate: 2023-07-08
+Installers:
+  - Architecture: x64
+    InstallerType: zip
+    NestedInstallerType: portable
+    InstallerUrl: https://github.com/TranslucentTB/TranslucentTB/releases/download/2023.1/TranslucentTB-portable-x64.zip
+    InstallerSha256: 5D2B8582A105003B4706DCA23F7D4080CD238F0BAB23065800DDD85E09298D85
+    MinimumOSVersion: 10.0.22621.0
+    Dependencies:
+      PackageDependencies:
+        - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+    NestedInstallerFiles:
+      - RelativeFilePath: TranslucentTB.exe
+        PortableCommandAlias: TranslucentTB
+  - Architecture: arm64
+    InstallerType: zip
+    NestedInstallerType: portable
+    InstallerUrl: https://github.com/TranslucentTB/TranslucentTB/releases/download/2023.1/TranslucentTB-portable-arm64.zip
+    InstallerSha256: BA5213D9FF7B60489BE80FDB58A357F6D83DD02F1FF7BCF25C71A3CC3E10FC0F
+    MinimumOSVersion: 10.0.22621.0
+    Dependencies:
+      PackageDependencies:
+        - PackageIdentifier: Microsoft.VCRedist.2015+.x64

```

Comment:
This is wrong.

Reply:
According to the Microsoft's docs
![image](https://github.com/microsoft/winget-pkgs/assets/96709586/535e27b2-96bb-4ea6-9f45-ddf1b5107f09)

Reply:
Just use the ARM64 redist.